tests/lib/snaps/store/test-snapd-layout-change-with-daemon/README.md

```diff
@@ -0,0 +1,11 @@
+# Snap build instructions
```

Comment:
we have a dedicated publisher for test snaps, we also try often to avoid store snaps when not needed but I see you already created the snaps, @sergiocazzolato  can help with this

Reply:
Yeah, that's only an "internal" comment, to explain to other developers how to build them if needed. I had experience with this because of another MR (the one that creates XDG_RUNTIME_DIR). I'll ask @sergiocazzolato to take ownership of these two.